### ISSUE TITLE
Format button and autocomplete checkbox should be always displayed

### ISSUE THREAD

**Issue by madalincm** [mozilla/redash#344]
### Steps to Reproduce:
1. Open new query page: https://pipeline-sql.stage.mozaws.net/queries/new
2. Observe the layout.

### Expected results:
The buttons are always displayed, the user has an easy access to all of them.

### Actual results:
The user needs to hover inside the section where he writes the query for the Format button and autocomplete checkbox to be displayed

### Notes/Issues:
Build identifier: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:58.0) Gecko/20100101 Firefox/58.0
Verified in FF58(Win7).  Issue can be reproduced in STMO-stage
![uichangesformatathover](https://user-images.githubusercontent.com/15685441/36735243-61989de8-1bde-11e8-86af-5466babb11ac.gif)

**Comment by alison985** [comment]
This is reproducible on https://pipeline-sql.stage.mozaws.net/ today.

**Comment by alison985** [comment]
Related to https://github.com/getredash/redash/issues/2644

**Comment by kocsmy** [comment]
I'm just wondering: is the auto-complete checkbox something that needs to be visible all the time (or more like a global settings that will be toggled once in a while)?

I'm wondering how could we find a place in the interface that can scale well with future improvements without over-populating the editor area.

**Comment by kocsmy** [comment]
If this will be a frequently used feature, it can go next to the new Parameters and Format buttons below the editor in https://github.com/getredash/redash/issues/2644

**Comment by kocsmy** [comment]
If it's a less important, maybe we'll need to introduce a new dropdown menu close to the editor where we can add things like this and even more later.

**Comment by alison985** [comment]
@jezdez FYI about the comment above.

**Comment by jezdez** [comment]
@alison985 Thank you!

@kocsmy Excellent points, scaling this high-density UI is tricky and is very likey to become more complicated the more the Redash query editor grows.

Especially with plans to add more support for frontend extensions (similar to the backend extensions) that our custom autocomplete checkbox could use, I would say the area next to the new Parameters and Format buttons like in https://github.com/getredash/redash/issues/2644 is hard to handle without adding an overflow technique like a dropdown. That said maybe we can table the scaling issue of that new area till a later date and just deal with the currently existing features.

So in short: +1 on your idea to implement it like in https://github.com/getredash/redash/issues/2644.

**Comment by jezdez** [comment]
https://github.com/getredash/redash/pull/2709 has moved the format button to the new area below the query textarea following https://github.com/getredash/redash/issues/2644.

I'm closing this now since porting the autocomplete toggle to upstream is tracked in https://github.com/mozilla/redash/issues/460.

**Comment by vcarciu** [comment]
Verified as fixed : 
![format](https://user-images.githubusercontent.com/1583842/45229542-c0a75900-b2ce-11e8-9c45-e61eb6f70173.gif)
